cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/model/Broker.java

```diff
@@ -75,6 +75,10 @@ public Host host() {
     return _host;
   }
 
+  public State getState() {
```

Comment:
Nit: usually we just call `state()` without the "get" if it is just an accessor but not doing any real work.